skrf/calibration/tests/test_calibration.py

```diff
@@ -70,6 +68,47 @@ def test_determine_reflect(self):
 
         [ self.assertEqual(k,l) for k,l in zip(self.r, r_found)]
 
+    def test_determine_reflect_matched_thru_and_line_ideal_reflect(self):
+        freq = rf.F(.25, .7, 21, unit = 'GHz')
+        medium=Coaxial.from_attenuation_VF(freq, att = 3.0, VF = .69)
+
+        thru = medium.line(0, 'm')
+        line = medium.line(0.12, 'm')
+
+        short = rf.two_port_reflect(medium.short(), medium.short())
+        r = determine_reflect(thru, short, line)
+        npy.testing.assert_array_almost_equal( r.s, -npy.ones_like(r.s))
+
+        reflect = rf.two_port_reflect(medium.open(), medium.open())
+        r = determine_reflect(thru, reflect, line, reflect_approx = medium.open())
+        npy.testing.assert_array_almost_equal(r.s, npy.ones_like(r.s))
+
+    def test_determine_reflect_matched_thru_and_line(self):
+        freq = rf.F(.25, .7, 40, unit = 'GHz')
+        medium = Coaxial.from_attenuation_VF(freq, att = 3.0, VF = .69)
+
+        thru = medium.line(0, 'm')
+        line = medium.line(0.12, 'm')
+        short = medium.short()
+
+        rng = npy.random.default_rng(12)
+        short.s[:, 0, 0] = short.s[:, 0, 0]+rng.uniform(-.02, 0.02, freq.f.size) + rng.uniform(-.02, 0.02, freq.f.size)*1j
+
+        r = determine_reflect(thru, rf.two_port_reflect(short, short), line)
+        npy.testing.assert_array_almost_equal( r.s, short.s)
+
+
+    def test_determine_reflect_regression(self):
```

Comment:
What is this test for? Seems very unclear when it's full of unexplained numbers.

Reply:
I modified the test so it is a bit clearer (this test is failing on current master, see comment above)